src/app/components/game/follow-widget/follow-widget.ts

```diff
@@ -110,4 +124,10 @@ export default class AppGameFollowWidget extends Vue {
 			}
 		}
 	}
+
+	onFollowPopoverDismissed() {
+		if (!this.game.developer.is_following) {
+			UserFollowSuggestion.dontSuggest(this.game.developer.id);
```

Comment:
I don't like using words that should have an apostrophe. Call `noSuggest` or `doNotSuggest` or something else